### DISCUSSION TITLE
The ability to assign all controls and buttons in Playtime 2 to CC commands

### DISCUSSION THREAD

**Discussion by Akognito** [helgoboss/helgobox#1266] [Ideas]
The ability to assign all controls and buttons in Playtime 2 to CC commands. It is very necessary for a live performance

**Comment by Akognito** [comment]
There are functions, but it is impossible to use them, and I would like to get a big adaptation of the interface for touch screens

**Comment by RibbleFripFnibbulator** [comment]
Yes please!

**Comment by helgoboss** [comment]
Duplicate of #978.
